learn/cookbooks/docker.md

```diff
@@ -0,0 +1,123 @@
+
+Docker containers are distributed in images. To use Meilisearch, use the `docker pull` command to download a Meilisearch image:
+
+```sh
+docker pull getmeili/meilisearch:v0.28
+```
+
+Meilisearch deploys a new Docker image for every new release. All images are tagged with the release version—indicated in the example above by the text following the `:`. You can see [the full list of available Meilisearch Docker images](https://hub.docker.com/r/getmeili/meilisearch/tags#!) on Docker Hub.
+
+::: warning
+The `latest` tag will always download the most recent Meilisearch release. Meilisearch advises against using it, as it might result in different machines running different images if significant time passes between setting up each one of them.
+:::
+
+## Run Meilisearch
+
+After running `docker pull` and downloading the Meilisearch image, use `docker run` to launch it:
+
+```sh
+docker run -it --rm \
+  -p 7700:7700 \
+  -v $(pwd)/meili_data:/meili_data \
+  getmeili/meilisearch:v0.28
+```
+
+### Configure Meilisearch
+
+Meilisearch accepts a number of instance options during launch. You can configure instance options in two ways: environment variables and CLI arguments. Some options are only available as CLI arguments—[consult our configuration reference for an exhaustive list](/learn/configuration/instance_options.md).
+
+#### Passing instance options with environment variables
+
+To pass environment variables to Docker, you must pass the `-e` argument to `docker run`. The example below launches Meilisearch with a master key:
+
+```sh
+docker run -it --rm \
+  -p 7700:7700 \
+  -e MEILI_MASTER_KEY='MASTER_KEY'\
+  -v $(pwd)/meili_data:/meili_data \
+  getmeili/meilisearch:v0.28
+```
+
+#### Passing instance options with CLI arguments
+
+If you want to pass command-line arguments to Meilisearch with Docker, you must explicitly run the `meilisearch` binary after indicating the Meilisearch image you want to use:
+
+```sh
+docker run -it --rm \
+  -p 7700:7700 \
+  -v $(pwd)/meili_data:/meili_data \
+  getmeili/meilisearch:v0.28
+  meilisearch --master-key="MASTER_KEY"
+```
+
+## Managing data
+
+When using Docker, your working directory is `/meili_data`. This means the location of your database file is `/meili_data/data.ms`.
+
+### Data persistency
+
+By default, data written to a Docker container is deleted every time the container stops running. This data includes your indexes and the documents they store.
+
+To keep your data intact between reboots, specify a dedicated volume by running Docker with the `-v` command-line option:
+
+```sh
+docker run -it --rm \
+  -p 7700:7700 \
+  -v $(pwd)/meili_data:/meili_data \
+  getmeili/meilisearch:v0.28
+```
+
+The example above uses `$(pwd)/meili_data`, which is a directory in the host machine. Depending on your OS, mounting volumes from the host to the container might result in performance loss and is only recommended when developing and prototyping your application.
```

Comment:
Streamlining since this is slightly redundant

```suggestion
The example above uses `$(pwd)/meili_data`, which is a directory in the host machine. Depending on your OS, mounting volumes from the host to the container might result in performance loss and is only recommended when developing your application.
```